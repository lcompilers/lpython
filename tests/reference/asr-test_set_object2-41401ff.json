{
    "basename": "asr-test_set_object2-41401ff",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_set_object2.py",
    "infile_hash": "e7360eff7caf0991c5bd4c505a947d23e2bc01277e9a2966362400df",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_set_object2-41401ff.stderr",
    "stderr_hash": "4fe845a8f949fce5b955b86d5a5ad60f0e1ae84e3c17b01572d37e2a",
    "returncode": 2
}