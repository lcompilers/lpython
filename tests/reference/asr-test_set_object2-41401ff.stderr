semantic error: Unhashable type: 'dict'
 --> tests/errors/test_set_object2.py:4:17
  |
4 |     my_set: set[dict[i32, str]] = {{1: "a", 2: "b"}, {3: "c", 4: "d"}}
  |                 ^^^^^^^^^^^^^^ Mutable type 'dict' cannot be stored in a set.
